Assets/VRTK/Scripts/Utilities/ObjectFollow/VRTK_TransformFollow.cs

```diff
@@ -22,45 +22,66 @@ public enum FollowMoment
         }
 
         [Tooltip("The moment at which to follow.")]
-        public FollowMoment moment = FollowMoment.OnPreRender;
+        [SerializeField]
+        private FollowMoment _moment = FollowMoment.OnPreRender;
```

Comment:
This should probably be protected too in case of wanting to inherit and modify the script